src/refactorings/merge-if-statements/index.ts

```diff
@@ -13,11 +15,21 @@ const config: RefactoringWithActionProvider = {
   actionProvider: {
     message: "Merge if statements",
 
-    canPerform(ast: t.AST, selection: Selection) {
-      const { mergeAlternate, canMerge } = tryMergeIfStatements(ast, selection);
-      this.message = mergeAlternate ? "Merge else-if" : "Merge if statements";
+    canPerformVisitorFactory: canMergeStatementsVisitorFactory,
 
-      return canMerge;
+    canPerformRefactoringMutator(
```

Comment:
Not 100% sold on the name yet (not sure what "mutator" refers to, regarding the function output).